modules/migration-provided-metrics.adoc

```diff
@@ -38,7 +38,7 @@ This metric is a cumulative count of Kubernetes API requests that `MigrationCont
 |Queryable label name |Sample label values |Label description
 
 |cluster
-|`\https://migcluster-url:443`
+|`\https://www.migcluster.com:443`
```

Comment:
See OpenShift guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#example-urls

Reply:
@apinnick 
Do you mean that it should be literally \https://www.example.com? I thought the word 'example' should be replaced with something meaningful.

Reply:
Yes. "migcluster" means nothing to the end user. You can't make it any more meaningful.